### ISSUE TITLE
Multitenancy support

### ISSUE THREAD

**Issue by Malekhourani** [osTicket/osTicket#6508]
Is there any easy way to add multitenancy support to OSTicket without doing changes manually?

**Comment by JediKev** [comment]
@Malekhourani 

What exactly do you mean? Like multi-brand where you have different branded frontends depending on the URL? Or running multiple instances of osTicket running on one server?

Cheers.

**Comment by Malekhourani** [comment]
> Like multi-brand where you have different branded frontends depending on the URL

Yes exactly, i want the same instance to handle requests for different brands, and each one of them will have its own database.

**Comment by JediKev** [comment]
@Malekhourani 

That will require some customizations and extra work on the webserver/database end (for routing, etc.). We do not support this out-of-the-box at this time. This is something on our Feature Request list for possible future development so please stay tuned!

Cheers.
